libs/components/tabs/src/lib/modules/tabs/tabset-nav-button-type.ts

```diff
@@ -1,4 +1,4 @@
 /**
- * Specifies the nav button type. The valid options are `previous`, `next`, and `finish`.
+ * The nav button type. The valid options are `previous`, `next`, and `finish`.
```

Comment:
```suggestion
 * The nav button type.
```
I don't see this text in the tab docs. Should it be? If so, should we file a bug?

Reply:
@johnhwhite, do we need a bug because this doesn't show up in the docs?

Reply:
I've filed AB#2442462 to follow up on this one.